src/Reflection/ReflectionProperty.php

```diff
@@ -286,6 +286,7 @@ public function isInitialized(object|null $object = null): bool
                 return false;
             }
 
+            // @infection-ignore-all Throw_: False positive, it's tested by testIsInitializedThrowsError()
```

Comment:
Related: https://github.com/sebastianbergmann/php-code-coverage/pull/969 ?

Reply:
Yes, it looks it's related. So this PR can probably wait for the release :)

Reply:
Probably worth doing that, ya :+1:

Reply:
Butfix released, a rebase now should give 100% MSI